website/www/site/content/en/documentation/patterns/batch-elements.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Pattern for dynamically grouping elements"
+---
+
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Dynamically grouping elements using the `BatchElements`-transform
+
```

Comment:
This looks great! Can we add this to https://beam.apache.org/documentation/patterns/overview/?

Reply:
done